v1/examples/core-cri-authz-request.json

```diff
@@ -2,7 +2,7 @@
   "aud": "http:\/\/www.example.com\/audience",
   "sub": "test-user-id",
   "nbf": 1659007190,
-  "shared_claims": {
+  "claims": {
```

Comment:
Nope, it's called `shared_claims` per https://github.com/alphagov/digital-identity-architecture/blob/main/rfc/0031-ipv-generic-frontend-cri-interface.md#requests-to-the-authorization-endpoint

(I didn't like this, but it's not the same format as an [OpenID Connect `claims`](https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter) so it's fine that it be called something different. I might have preferred to call it `credentialSubject` maybe.)

Reply:
OK cool my misunderstanding, so we use `shared_claims` internally to pass data, but what we get from a Relying Party is an OpenID Connect `claims` value to let us know what they expect back from the `/userinfo` endpoint, so we need to support both as optional things?  Once added I could another test .json file for a sample external relying party request?

Reply:
Ah well this file is *CRI* authz request, so if we're documenting the RP interface that'd be somewhere else.

Reply:
Yeah I will modify the underlying AuthorizationRequestClass, so it can support both `shared claims` and `claims` and introduce a new test file rp-authz-request.json and make that reflect what the RP needs to send to the authorize endpoint.  This should hopefully make the AuthorizationRequestClass generic enough that it can be used for both purposes.

Reply:
Updates added to put shared_claims back in and added support for claims from a Relying Party call to the authorize endpoint